### PR TITLE
Fixing API Reference Docs for 1.0

### PR DESCRIPTION
**Pull request by AnthonyMDev**

### CODE REVIEW

**Comment by AnthonyMDev** [comment]
@trevorblades Trying to regenerate some of our API docs for the 1.0 release. The deploy preview link is giving me a 404. Any idea why that is?

Do I need to base the commit on `main` for docs to deploy properly? We've been working off a release branch that will eventually get merged into `main`.

**Comment by trevorblades** [comment]
Yes, since the `release/1.0` branch contains no `netlify.toml` file, Netlify doesn't know what to build or how to build it. Here are the deploy logs for this PR: https://app.netlify.com/sites/apollo-ios-docs/deploys/62cc73d8331b38000894dddc

I noticed it says `12:03:05 PM: No build steps found, continuing to publishing`

Switching the base to main would resolve this, or you could add the same `netlify.toml` to the `release/1.0` branch and rebase this PR.

**Comment by AnthonyMDev** [comment]
Okay thanks,. I'll add the `netlify.toml` to the release branch!

File: Sources/Apollo/Documentation.docc/Index.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+@Metadata {
+   @TechnologyRoot   
+}
+
+API Reference Documentation for Apollo iOS.
+
+## Overview
+
+Our API reference is automatically generated directly from the inline comments in our code. If you're contributing to Apollo, all you have to do is add inline documentation comments and they will appear here.
```
Comment (calvincestari) [inline Sources/Apollo/Documentation.docc/Index.md:11]: Is it worth telling people we now use DocC and to follow those guidelines?

```suggestion
Our API reference is automatically generated directly from the inline comments in our code. If you're contributing to Apollo, all you have to do is add inline documentation comments, conforming to [DocC formatting guidelines](https://developer.apple.com/documentation/xcode/writing-symbol-documentation-in-your-source-files), and they will appear here.
```

File: Sources/Apollo/Documentation.docc/Index.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+See something missing documentation? Add inline documentation comments to the code, and open a pull request!
```
Comment (calvincestari) [inline Sources/Apollo/Documentation.docc/Index.md:13]: nit:

```suggestion
See something missing in the documentation? Add inline documentation comments to the code, and open a pull request!
```

File: netlify.toml [hunk 3]
```diff
@@ -10,7 +10,7 @@
   command = """\
   cd ../
   rm -rf monodocs
-  git clone https://github.com/apollographql/docs --branch release/1.0 --single-branch monodocs
+  git clone https://github.com/apollographql/docs --branch main --single-branch monodocs
```
Comment (calvincestari) [inline netlify.toml:13]: Is this targeting of `main` instead of `release/1.0` correct? cc. @trevorblades

**Review by calvincestari — APPROVED** [review]
This looks great, really nice infrastructure getting built here. Awesome work - than you!

File: netlify.toml (hunk 3 above)
Comment (AnthonyMDev) [reply netlify.toml:13]: Yeah, I made the same mistake here. This is cloning the docs repo not ours!

**Comment by AnthonyMDev** [comment]
Thanks for the suggestions @calvincestari!

**Pull request merged**